### PR TITLE
Update SubSystems NuGet packages, except for masstransit

### PR DESCRIPTION
**Pull request by sahma19**

### CODE REVIEW

File: domains/authorization/Authorization.API/API/Program.cs [hunk 1]
```diff
@@ -14,9 +14,9 @@
 var otlpConfiguration = builder.Configuration.GetSection(OtlpOptions.Prefix);
 var otlpOptions = otlpConfiguration.Get<OtlpOptions>()!;
 
-builder.AddSerilogWithOpenTelemetry(otlpOptions.ReceiverEndpoint);
+builder.AddSerilog();
```
Comment (tnickelsen) [inline domains/authorization/Authorization.API/API/Program.cs:17]: Don't use setup anymore?

File: domains/transfer/ClaimAutomation/Worker/Program.cs [hunk 2]
```diff
@@ -26,7 +26,7 @@
 var otlpConfiguration = builder.Configuration.GetSection(OtlpOptions.Prefix);
 var otlpOptions = otlpConfiguration.Get<OtlpOptions>()!;
 
-builder.AddSerilogWithOpenTelemetry(otlpOptions.ReceiverEndpoint);
+builder.AddSerilog();
```
Comment (tnickelsen) [inline domains/transfer/ClaimAutomation/Worker/Program.cs:29]: Same here

File: domains/authorization/Authorization.API/API/Program.cs (hunk 1 above)
Comment (sahma19) [reply domains/authorization/Authorization.API/API/Program.cs:17]: Yes, But the Method in the setup package was changed, in version 4.0.0, to no longer export logs to opentelemetry. They will now be scraped by PromTail

File: domains/transfer/ClaimAutomation/Worker/Program.cs (hunk 2 above)
Comment (sahma19) [reply domains/transfer/ClaimAutomation/Worker/Program.cs:29]: Same answer

**Review by tnickelsen — APPROVED** [review]

**Pull request merged**